_posts/2019-04-16-sicko-mode.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "\"Sicko\" Mode: a modern critique of homophobia in America"
+author: Michael Luo
+date:   2019-04-16
+categories: drake travis scott michael
```

Comment:
I think your name should not be a topic

Reply:
I think eventually we may wish to allow users to filter by their favourite author.

Reply:
That is available through the field `author`

Reply:
Perhaps.